config/default/kustomization.yaml

```diff
@@ -17,49 +17,13 @@ bases:
 - ../rbac
 - ../manager
 - ../prometheus
-# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in
-# crd/kustomization.yaml
-#- ../webhook
-# [CERTMANAGER] To enable cert-manager, uncomment all sections with 'CERTMANAGER'. 'WEBHOOK' components are required.
-#- ../certmanager
-# [PROMETHEUS] To enable prometheus monitor, uncomment all sections with 'PROMETHEUS'.
-#- ../prometheus
+# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in crd/kustomization.yaml
+- ../webhook
 
-# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in
-# crd/kustomization.yaml
-#- manager_webhook_patch.yaml
+# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in crd/kustomization.yaml
+patchesStrategicMerge:
+  - manager_webhook_patch.yaml
+  - webhookcainjection_patch.yaml
```

Comment:
Do we need those? Seems like they are related to the certificate.

Reply:
So, after discussing it, OCP only contains an equivalent to cert-manager and instead of applying `Certificate` objects we just add the `service.beta.openshift.io/serving-cert-secret-name: <secret name>` annotation to the webhook service.

We still need to mount the secret to the webhook pod (KMM pod) so those are still needed here as well.